CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Adobe Code of Conduct
```

Comment:
nit: there is a newer version of this file in the OSS repo that adds some punctuation. no need to update i think as the oss team didn't block my repo request when using this version.

Reply:
Yeah, let's take it up during extension hardening.